index.js

```diff
@@ -213,8 +241,8 @@ class CoverageSonar extends CoverageBase {
      * @return {Promise}                   An access token that build can use to talk to coverage server
      */
     _getAccessToken(buildCredentials) {
-        const { jobId } = buildCredentials;
-        const projectKey = `job:${jobId}`;
+        const { jobId, pipelineId } = buildCredentials;
+        const projectKey = getProjectKey({ enterpriseEnabled: sonarEnterprise, jobId, pipelineId });
         const username = `user-job-${jobId}`;
```

Comment:
We don't need separate users for each job, so this should also be `pipelineId`

Reply:
updated